packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel/test/Providers/Abstractions/ClientPipelineApiTests.cs

```diff
@@ -28,7 +29,7 @@ public void ValidateBodyOfRestClientOperationIsOverridden()
 
             Assert.NotNull(method);
             Assert.NotNull(method!.BodyStatements);
-            var test = method?.BodyStatements?.ToDisplayString();
+            var test = method!.BodyStatements!.ToDisplayString();
```

Comment:
is this used? If not can we please remove it?

Reply:
remvoed